Gruntfile.js

```diff
@@ -18,79 +18,38 @@ module.exports = function(grunt) {
         'm2': 'http://localhost:9003/m2/'
       },
       'test': {
-        'm1': 'https://test.openolitor.ch/m1/'
+        'm1': 'https://sunu-server-test.scapp.io/sunu/'
```

Comment:
Stop! This can't be replaced. We need to find a way to cover sunu and oo-switzerland!

Reply:
done